libs/zzz/page-optimize/src/BonusStats.tsx

```diff
@@ -42,8 +42,7 @@ export function BonusStatsSection() {
   const { database } = useDatabaseContext()
   const { key: characterKey } = useCharacterContext()!
   const { bonusStats } = useCharOpt(characterKey)!
-  const charMetaDesc =
-    characterKey && database.charMeta.get(characterKey).description
+  const [charMetaDesc, setCharMetaDesc] = useState<string | undefined>('')
```

Comment:
use `useDataManagerBase(database.charMeta, key)` since that already has a state and triggers on update.